Replace QuickMenuNewButton to QuickMenuMiddleButton

codeimpl opened the pull request:
QuickMenuNewButton has been implemented, and this patch rename it
because it is easier to understand the layout by renaming it to
QuickMenuMiddleButton.

Refs: #983

romandev commented on src/org/triple/banana/quick_menu/QuickMenuDialog.java:
```diff
@@ -63,8 +61,8 @@ private void updateButtons(@NonNull List<ButtonInfo> buttons, @NonNull View.OnCl
             button.setId(info.id);
-            button.setImageResource(info.image);
-            button.setLabel(info.label);
+            button.setCompoundDrawablesWithIntrinsicBounds(0, info.image, 0, 0);
```
Add a `setIcon()` method and then use it.

codeimpl replied:
done.

romandev commented on src/org/triple/banana/quick_menu/QuickMenuDialog.java:
```diff
@@ -63,8 +61,8 @@ private void updateButtons(@NonNull List<ButtonInfo> buttons, @NonNull View.OnCl
         for (final ButtonInfo info : buttons) {
             final QuickMenuButton button = new QuickMenuButton(getContext());
```
QuickMenuNewButton?

codeimpl replied:
As we discussed, QuickMenuMiddleButton is better.

romandev commented on BUILD.gn:
```diff
@@ -198,7 +198,6 @@ android_library("triple_banana_java") {
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuDialog.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactory.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java",
-    "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuNewButton.java",
```
QuickMenuIconButton? or better name?

codeimpl replied:
As we discussed, QuickMenuMiddleButton is better.

romandev commented on BUILD.gn:
```diff
@@ -198,7 +198,7 @@ android_library("triple_banana_java") {
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuDialog.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactory.java",
     "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuFactoryImpl.java",
-    "//triple_banana/src/org/triple/banana/quick_menu/QuickMenuNewButton.java",
```
You should have remove this file actually.

codeimpl replied:
sure~ thanks!

romandev reviewed (commented):
Please update your commit description as per your actual change.
The commit description should be updated in commit side and then PR description also should be updated manually.

codeimpl commented:
> Please update your commit description as per your actual change.
> The commit description should be updated in commit side and then PR description also should be updated manually.

Sure, I updated commit description, but PR description doesn't. So I updated both of them. Thanks.

romandev reviewed (approved):
lgtm
